src/models/userModel.ts

```diff
@@ -226,16 +227,32 @@ export class UserModel extends Model {
         });
     }
 
+    public checkUserCompany
+    (userID : string, companyID : string) : Promise<Object> {
+        return new Promise((resolve : (data : Object) => void,
```

Comment:
La funzione `resolve` dichiara `Object` ma in linea `240` viene risolta con un `boolean`. Corretto?